components/data_comps/drof/cime_config/namelist_definition_drof.xml

```diff
@@ -177,6 +179,7 @@
       <value stream="rof.diatren_iaf_ais00_rx1">$DIN_LOC_ROOT/lnd/dlnd7/RX1</value>
       <value stream="rof.diatren_iaf_ais45_rx1">$DIN_LOC_ROOT/lnd/dlnd7/RX1</value>
       <value stream="rof.diatren_iaf_ais55_rx1">$DIN_LOC_ROOT/lnd/dlnd7/RX1</value>
+      <value stream="rof.iaf_jra_1p5"          >$DIN_LOC_ROOT/ocn/jra55/v1.5_noleap</value>
```

Comment:
@darincomeau - any reason not to put these files in the lnd/dlnd7 directory?

Reply:
yes - these files were all together in one directory on the ccsm svn inputdata server https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ocn/jra55/v1.5_noleap/, and I set this path so it would automatically download them. I meant to make a point here that the directory structure is different than for previous versions.

I'd be happy to move the `drof` files though so they sit in a directory next to the v1.4 counterparts. One other consideration is that these files may (hopefully) be updated as new years become available, so I thought to have our directory structure match the NCAR one.

Reply:
Oooo, matching NCAR is a dangerous rationale! I guess it seems more natural for these files to be co-located with the streams domain file, which at this point is in dlnd. But it might be worth a conversation at some point if datm files should be stored with the atm and drof files with the rof, and not in either the ocn or lnd?

Reply:
Sure, happy to discuss re-organizing this. Thanks again for pointing out, I meant to flag this in a comment.

Reply:
Ah, any re-organizing should probably be done in the future. I think as long as you document the reasons for the files being where they are this PR is fine as-is